examples/websocket-retry-client/websocket_retry_client.description

```diff
@@ -0,0 +1,5 @@
+// If the WebSocket client lost a connection by the server being shut down or by getting any IOExceptions,
+// it automatically reconnects to the same backend until the connection becomes successful or until
```

Comment:
How about something like this ?

`If the WebSocket client lost the connection due to some transient failure, it automatically tries to reconnect to the given backend. If the maximum reconnect attempts is reached it gives up on the connection.`

Reply:
Fixed with ce5d648